hardhat.config.ts

```diff
@@ -1,5 +1,5 @@
 import * as dotenv from "dotenv";
-import "ethers-maths";
+import "evm-maths";
```

Comment:
What's the difference ?

Reply:
It is the same package but this one is independent from ethers and only uses bigint. It also adds the `expN` function which helps calculate `wTaylorCompounded`

Reply:
It is only about showcasing that ethers-maths is deprecated now and that evm-maths is the new package to use